lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -298,13 +298,13 @@ def self.get_values(json, category_type, override_price = false)
 
     # [MB/s capacity, Ops/s capacity]
     PER_SEC_CAPACITIES = {
-      'c1.medium' => [6000, 300000],
-      'c1.xlarge' => [6000, 300000],
-      'c3.2xlarge' => [8000, 600000],
-      'c3.4xlarge' => [9000, 600000],
-      'c3.8xlarge' => [15000, 600000],
-      'c3.large' => [5000, 600000],
-      'c3.xlarge' => [7000, 600000],
+      'c1.medium' => [118, 5471],
+      'c1.xlarge' => [347, 17245],
+      'c3.2xlarge' => [947, 63297],
+      'c3.4xlarge' => [962, 96546],
+      'c3.8xlarge' => [964, 60702],
```

Comment:
Is it worth "normalizing" values that are super close in a given family? I mean, it looks too good to be a coincidence here :-)
What do you both think?